Core/src/IfcGeometryConverter/ProfileConverter.h

```diff
@@ -1393,6 +1331,18 @@ namespace OpenInfraPlatform {
 				}
 			}
 
+			void SearchExistingVertex(std::stringstream ID, uint32_t index, carve::geom::vector<3> position) const
+			{
+				auto itFound = polygonIndices.find(ID.str());
+				if (itFound != polygonIndices.end()) {
+					index = itFound->second;
+				}
+				else {
+					index = polygon->addVertex(position);
+					polygonIndices[ID.str()] = index;

```

Comment:
Does this compile? `polygonIndices` is not defined in scope ...

Reply:
However it compiles :)

Reply:
Resolved: 1b6fe62

Reply:
This means that the calling function is not called anywhere. Need to dig deeper into that.